larq/quantized_variable_test.py

```diff
@@ -299,3 +300,19 @@ def f():
             assert evaluate(x) == -2.0
     else:
         assert evaluate(x) == 0.0
+
+
+@pytest.mark.skipif(
+    version.parse(tf.__version__) < version.parse("2"), reason="Requires TensorFlow 2",
+)
+@pytest.mark.skipif(
+    version.parse(tf.__version__) == version.parse("2.3.0rc0"),
+    reason="Currently broken on 2.3.0rc0.",
```

Comment:
Should we make an issue tracking the TF 2.3 release to keep track of potential blockers/things to fix like this?

Reply:
:+1: I will look into this soon and will either make an issue or provide a fix.

Reply:
2c30f2170cc84e8cd839d59018dbf865a0745f62 should fix the incompatibility with TF 2.3